src/unistd.rs

```diff
@@ -756,12 +756,29 @@ pub fn close(fd: RawFd) -> Result<()> {
     Errno::result(res).map(drop)
 }
 
+/// Reads from a raw file descriptor.
+/// Reads at most as many bytes as will fit into the provided buffer.
+///
+/// Returns the number of bytes successfully read.
```

Comment:
Put this in the same paragraph as the above sentence and please rewrite it so it flows a little better. Something like "The number of bytes actually read is returned."

Reply:
"The number of bytes actually read is returned." does not flow better in my opinion. Passive voice not necessary here.

Reply:
Sure, I'm not too opinionated about this.